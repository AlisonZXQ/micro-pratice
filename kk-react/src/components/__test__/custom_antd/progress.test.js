import React from 'react';
import { shallow } from 'enzyme';
import mountTest from '@tests/shared/mountTest';
import Progress from '../../CustomAntd/progress';

describe('progress', () => {
  mountTest(Progress);

  it('should snapshot correctly', () => {
    const shallowWrapper = shallow(<Progress />);
    expect(shallowWrapper).toMatchSnapshot();
  });
})

