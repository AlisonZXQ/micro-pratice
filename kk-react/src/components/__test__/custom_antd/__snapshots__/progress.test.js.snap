// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`progress should snapshot correctly 1`] = `
<span
  className="container"
>
  <Progress
    gapDegree={0}
    percent={0}
    showInfo={true}
    size="small"
    strokeColor={Object {}}
    strokeLinecap="round"
    trailColor="#f3f3f3"
    type="line"
  />
</span>
`;
